year-pages/2019-conferences-list.md

```diff
@@ -3,11 +3,13 @@
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
 | [React India](https://www.reactindia.io/) | 22-24 Aug | Goa | A 3 day conference with first day focussing on workshops & the next 2 days talks on things around React, React Native, GraphQL, VR & AR, Redux, Preact and more | No |
+| [Agile India 2019](https://2019.agileindia.org/) | 17th - 23rd Mar | Sheraton Grand Bangalore Hotel, Bengaluru | Asia's Largest and Premier International conference on Leading Edge Software Development Methods | No |
 | [ReactFoo](https://reactfoo.in/2019/) | 1-2 Mar | NIMHANS Convention Centre, Bengaluru | A conference on React, alternatives to React, React Native and front-end engineering | No |
 | [SACON](https://www.sacon.io/) | 15-16 Feb | Bengaluru |  India’s 1st Security Architecture Conference | No |
 | [PGConf India](https://pgconf.in/conferences/pgconfin2019) | 13-15 Feb | Bengaluru | Conference for Postgresql users and devs | No |
 | [Processing Community Day](https://processingindia.org/bangalore.html) | 2 Feb| Bengaluru | Processing Community Day (PCD) is a day to celebrate and explore art, code, and diversity around the world, conducted by Processing community | No |
 | [RubyConf India](https://www.rubyconfindia.org/) | 20-21 Jan | Hotel Park Regis, Goa | A global event complementing other RubyConf events across the world. This year it's a single track 2 day event, focused on the Language, Framework and Tools | [Yes](https://www.rubyconfindia.org/scholarship/) |
 | [Pragma IIITA](http://pragmaconf.tech/) | 19 -20 Jan | Indian Insitute of Information Technology, Allahabad | Pragma is premier student design and developers’ conference held at IIIT-Allahabad. Attracting some of the best student developers from across India, Pragma is a 3 day extravaganza featuring a hackathon, workshops and numerous talks. | No
 | [GopherCon](https://gopherconindia.com/) | 18-19 Jan | Hotel Park Regis, Goa | Go conference in India. Go is an open source project developed by a team at Google and many contributors for the open source community | [Yes]( https://gopherconindia.com/scholarship/) |
+| [IN/Clojure](https://inclojure.org/) | 11-12 Jan | Royal Orchid Hotel, Bangalore | India's annual Clojure conference | No |
```

Comment:
s/Bangalore/Bengaluru 

It does have a scholarship: https://inclojure.org/#opgrant